src/neon.mk

```diff
@@ -0,0 +1,55 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := neon
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 0.30.2
+$(PKG)_CHECKSUM := db0bd8cdec329b48f53a6f00199c92d5ba40b0f015b153718d1b15d3d967fbca
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := http://webdav.org/$(PKG)/$(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc openssl expat gettext
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'http://webdav.org/$(PKG)/' | \
+    $(SED) -n 's,.*/\([0-9][^"]*\)/"\.tar.*,\1,p' | \
+    sort | uniq | \
+    head -1
+endef
+
+define $(PKG)_BUILD
+    cd '$(BUILD_DIR)' && \
+        ne_cv_fmt_size_t=%lu \
+        ne_cv_fmt_ssize_t=%lu \
+        ne_cv_fmt_off64_t=%I64u \
+        ne_cv_fmt_time_t=%lu \
+        ne_cv_libsfor_socket=-lws2_32 \
+        ne_cv_libsfor_gethostbyname=-lws2_32 \
+        '$(SOURCE_DIR)'/configure \
+              $(MXE_CONFIGURE_OPTS) \
+              $(MXE_DISABLE_DOCS) \
+              --with-ssl=yes
+    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
+    $(MAKE) -C '$(BUILD_DIR)' -j 1 install-lib install-headers install-nls
+
+    # create pkg-config file
+    $(INSTALL) -d '$(PREFIX)/$(TARGET)/lib/pkgconfig'
+    (echo 'prefix=$(PREFIX)/$(TARGET)'; \
+     echo 'exec_prefix=$${prefix}'; \
+     echo 'libdir=$${exec_prefix}/lib'; \
+     echo 'includedir=$${prefix}/include'; \
+     echo ''; \
+     echo 'Name: $(PKG)'; \
+     echo 'Version: $($(PKG)_VERSION)'; \
+     echo 'Description: neon is an HTTP and WebDAV client library'; \
+     echo 'Requires.private: openssl gettext'; \
```

Comment:
For me it complains as follows:

```
# create test binary
i686-w64-mingw32.static-gcc -W -Wall -Werror -std=c11 '/home/mxe/mxe-libcddb/src/neon-test.c' -o '/home/mxe/mxe-libcddb/usr/i686-w64-mingw32.static/bi
n/test-neon.exe' `i686-w64-mingw32.static-pkg-config neon --cflags --libs`
Package gettext was not found in the pkg-config search path.
Perhaps you should add the directory containing `gettext.pc'
to the PKG_CONFIG_PATH environment variable
Package 'gettext', required by 'neon', not found
/tmp/ccyTnSiE.o:neon-test.c:(.text+0x21): undefined reference to `ne_addr_resolve'
/tmp/ccyTnSiE.o:neon-test.c:(.text+0x37): undefined reference to `ne_addr_result'
/tmp/ccyTnSiE.o:neon-test.c:(.text+0x5a): undefined reference to `ne_addr_error'
/tmp/ccyTnSiE.o:neon-test.c:(.text+0x87): undefined reference to `ne_addr_first'
/tmp/ccyTnSiE.o:neon-test.c:(.text+0xaf): undefined reference to `ne_iaddr_print'
/tmp/ccyTnSiE.o:neon-test.c:(.text+0xce): undefined reference to `ne_addr_next'
/tmp/ccyTnSiE.o:neon-test.c:(.text+0xfa): undefined reference to `ne_addr_destroy'
collect2: error: ld returned 1 exit status
```

Can you show linking errors which you get without adding gettext, please?